frontend/test/metabase/scenarios/binning/correctness/longitude.cy.spec.js

```diff
@@ -11,7 +11,7 @@ const LONGITUDE_OPTIONS = {
   },
   "Bin every 1 degree": {
     selected: "1°",
-    representativeValues: ["167° W", "166° W", "60° W"],
```

Comment:
Had to update this, but it feels a bit weird. I would've expected there to be a "166° W" value in the chart based on these screenshots.

![image](https://user-images.githubusercontent.com/31325167/122954917-25e56f80-d380-11eb-9a7f-d8aa3cd90407.png)

![image](https://user-images.githubusercontent.com/31325167/122955324-4ad9e280-d380-11eb-8585-30f75015bf5a.png)